check-swlib.sh

```diff
@@ -13,33 +13,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
Can we please include the following block to be consistent with the other top-level *.sh script output logs:
```
echo Command used:
echo "$0 $@"
echo
```

Reply:
Done